clients/gtest/trsv_gtest.yaml

```diff
@@ -4,26 +4,26 @@ include: known_bugs.yaml
 
 Definitions:
   - &small_matrix_size_range
-    - { M:    -1, lda:     1 }
-    - { M:     4, lda:     4 }
-    - { M:    10, lda:    20 }
-    - { M:   128, lda:   256 }
+    - { M:    -1, lda:     1, stride_a: 1 }
+    - { M:     4, lda:     4, stride_a: 16 }
+    - { M:    10, lda:    20, stride_a: 300 }
+    - { M:   10, lda:   10, stride_a: 100 }
+    - { M:   128, lda:   128, stride_a: 32768 }
 
   - &medium_matrix_size_range
-    - { M:   192, lda:   192 }
-    - { M:   256, lda:   256 }
-    - { M:   384, lda:   384 }
-    - { M:   600, lda:   600 }
-    - { M:   800, lda:   801 }
+    - { M:   192, lda:   192, stride_a: 36864 }
```

Comment:
Not really, it just made the generation script cleaner since I could just add trsv to a previous case. Since it's defined in the same data set as it's requirements we aren't coming up with excessive permutations.